src/builder/modules/Generator.res

```diff
@@ -143,6 +147,72 @@ let drawLine = (
     Generator_Builder.drawLine(model.contents, from, to, ~color, ~width, ~pattern=[], ~offset=0)
 }
 
+let drawLinePath = (
+  points: array<Builder.position>,
+  ~color: string="#000000",
+  ~width: int=1,
+  /* ~pattern: array<int>=[],
+   ~offset: int=0, */
+  ~close: bool=false,
+  (),
+) => {
+  model :=
+    Generator_Builder.drawPath(
+      model.contents,
+      points,
+      ~color,
+      ~width,
+      ~pattern=[],
+      ~offset=0,
+      ~close,
+    )
+}
+
+let drawLineRect = (
```

Comment:
I'm guessing this one could be used to "section off" parts of a design? Maybe `drawLine()` would be good enough for that? And particular use cases you can think of for this one?